packages/teams-js/src/public/meeting.ts

```diff
@@ -538,6 +577,25 @@ export namespace meeting {
     ScreenShare = 'ScreenShare',
   }
 
+  /**
+   * Represents a validated email.
+   *
+   * @hidden
+   * Hide from docs.
+   */
+  export class EmailAddress {
```

Comment:
This class should have unit tests ensuring that incorrect email addresses are rejected.

Reply:
Sorry, forgot to add tests as I included additional logic, will add them